src/snowflake/snowpark/mock/_plan.py

```diff
@@ -1351,6 +1351,15 @@ def calculate_expression(
         try:
             return input_data[exp.name]
         except KeyError:
+            if exp.name.startswith('"') and exp.name.endswith('"'):
```

Comment:
It's better to process `exp.name` in the try block instead of adding a nested try block in the except case.

Reply:
love this idea, code updated